src/yukon/backend/src/uavcan_node_demo.py

```diff
@@ -58,19 +60,40 @@ class DemoApplication:
     """
 
     def __init__(self) -> None:
-        transport = pyuavcan.transport.udp.UDPTransport('127.0.0.42/8')
+        try:
+            self._transport = pyuavcan.transport.udp.UDPTransport(
+                '127.0.0.42/8')
+        except OSError as ex:
+            if (hasattr(errno, 'EADDRNOTAVAIL') and
+                    ex.errno == errno.EADDRNOTAVAIL):
```

Comment:
Are we not following the limit of 120 chars per line?

Reply:
Yes. And if it wasn't it would fail.